docs/src/main/paradox/bootstrap/kubernetes.md

```diff
@@ -0,0 +1,149 @@
+# Kubernetes
+
+As of Akka 2.5.15 and Akka-Management 0.18 the recommended way to run Akka Cluster in Kubernetes is to:
+
+* Use Akka Bootstrap with `akka-dns` with cluster formation via DNS SRV records
+* Use a headless service for internal for Akka management/bootstrap so that readiness probes for prod traffic don't interfere with bootstrap 
+* If required use a separate service and/or ingress for user facing endpoints e.g. HTTP gRPC
+
+# Kubernetes Services
+
+For Akka Cluster / Management use a headless service. This allows the solution to not be coupled to k8s as well
+as there is no use case for load balancing across management/remoting ports.
+Set endpoints to be published before readiness checks pass as these endpoints are required to bootstrap the Cluster
+and make the application ready. 
+
+
+```
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    appName: "akka-cluster-kubernetes"
+  annotations:
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
```

Comment:
really "true" or true would work too? for my info

Reply:
Annotations have to be strings :(

If you have it as true, `kubectl create` tells you it is invalid and `kubectl apply` ignores it so it doesn't get set (reported bug)